Modules/_xxsubinterpretersmodule.c

```diff
@@ -2025,7 +2025,7 @@ interp_create(PyObject *self, PyObject *args, PyObject *kwds)
     }
 
     // Create and initialize the new interpreter.
-    PyThreadState *save_tstate = PyThreadState_Swap(NULL);
+    PyThreadState *save_tstate = PyThreadState_Get();
     // XXX Possible GILState issues?
```

Comment:
Maybe we can delete this comment now?

Reply:
I'm not sure the gilstate issues with subinterpreters have been resolved.

Reply:
Going to leave it here